test/e2e/backup_test.go

```diff
@@ -0,0 +1,63 @@
+package e2e
+
+import (
+	"flag"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"golang.org/x/net/context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/vmware-tanzu/velero/pkg/cmd/cli/install"
+)
+
+var (
+	uuidgen              uuid.UUID
+	err                  error
+	backupName           string
+	restoreName          string
+	veleroInstallOptions *install.InstallOptions
+)
+
+func veleroInstall(cloudProvider string, useRestic bool) {
+	flag.Parse()
+	Expect(EnsureClusterExists(context.TODO())).To(Succeed(), "Failed to ensure kubernetes cluster exists")
+	Expect(err).To(Succeed(), "Failed to instantiate cluster client")
```

Comment:
This check can be removed as the cluster client isn't being instantiated here.

Reply:
Doh.. yes, removed.